lib/services.js

```diff
@@ -166,9 +166,12 @@ export default {
         const { objectKey, authInfo, size, contentMD5, metaHeaders,
             contentType, cacheControl, contentDisposition, contentEncoding,
             expires, multipart, headers, overrideMetadata, log,
-            lastModifiedDate } = params;
+            lastModifiedDate, versionId } = params;
         log.trace('storing object in metadata');
         assert.strictEqual(typeof bucketName, 'string');
+
+        const versioningParams = versionId ? { versionId } : {};
```

Comment:
They are different. Please refer to the API here https://github.com/scality/MetaData/pull/1009#issue-210629096.